src/common/helpers/record.helper.ts

```diff
@@ -111,3 +111,7 @@ export const getRecordDependencies = (record?: RecordEntry | null) => {
     };
   }
 };
+
+export const hasEmptyValues = (values: UserValueContents[]) => {
```

Comment:
Consider a name like `hasAllEmptyValues` or `hasOnlyEmptyValues`. The current name, to me, implies that some values are empty, not that all of them are.